### PR TITLE
drupal6: retain url_alias.dst as permalink in imported nodes

### PR DESCRIPTION
**Pull request by fl3a**
Hi there,
i think, it would be great to retain the Drupal url_alias as permalink in imported nodes.

so i...

- extended the query to be able to fetch url_alias.src

-  assigned permalink within Front Matter

Regards
Florian

### CODE REVIEW

**Review by DirtyF — DISMISSED** [review]
Some style issues left, you can launch tests locally with `script/cibuild`

**Comment by fl3a** [comment]
Fixed all script/cibuild code style offenses.

File: lib/jekyll-import/importers/drupal6.rb [hunk 1]
```diff
@@ -41,10 +43,12 @@ def self.post_data(sql_post_data)
         content = sql_post_data[:body].to_s
         summary = sql_post_data[:teaser].to_s
         tags = (sql_post_data[:tags] || "").downcase.strip
+        permalink = "/" + sql_post_data[:alias]
```
Comment (parkr) [inline lib/jekyll-import/importers/drupal6.rb]: This should new conditional, right? In case the data `sql_post_data[:alias]` isn't there?

**Review by parkr — APPROVED** [review]

**Comment by parkr** [comment]
@jekyllbot: merge +minor

**Pull request merged**